.github/workflows/bump.yml

```diff
@@ -48,21 +47,37 @@ jobs:
           passphrase: ${{ secrets.PHYLUM_BOT_GPG_PASSPHRASE }}
           git_user_signingkey: true
           git_commit_gpgsign: true
-          git_config_global: true
 
-      - name: Set up Homebrew
-        uses: Homebrew/actions/setup-homebrew@master
+      - name: Update URL and SHA256 values
+        id: commit
+        continue-on-error: true
```

Comment:
Why is this marked as `continue-on-error`? Is the goal to still create a PR and manually fix it? But the next check explicitly tests for failure, so that's probably not it.

Reply:
Nice catch. The `continue-on-error` logic was originally replicated from the `cli` and `phylum-ci` workflows for automatically updating dependencies. It was used there because there is a chance that there are NO dependencies that get updated...which would cause the `git` commands in that step to fail. `continue-on-error` is used for that situation so that the overall job does not show as a failure. In the case of this workflow, the files are explicitly modified with (presumably) a new/different CLI version and artifact SHA. Therefore, the `continue-on-error` logic has been taken out.